index.html

```diff
@@ -53,13 +53,13 @@
           companyURL: "https://github.com/melsumner",
           w3cid: 125865
         },
-        { name: "Michael Cooper",
+    ],
+    formerEditors: [
+    { name: "Michael Cooper",
```

Comment:
```suggestion
          { name: "Michael Cooper",
```

This is just a whitespace fix